C12.1_1.1

Constantine7266 opened the pull request:
Нужно ли что то изменить.

Constantine7266 committed:
C12.1_1.1

Nikolay38744 commented on Стр/10.1/12.1_1.py:
```diff
@@ -1,4 +1,5 @@
 # TODO выведи данные переменной четыре раза
-A = 9
\ No newline at end of file
+A = 9
+print(A * 4)
```
```suggestion
A = 9
print(A * 4)
```
```suggestion
B = "10"
print( B * 2)
```

Nikolay38744 commented on Стр/10.1/12.1_1.py:
```diff
@@ -1,4 +1,5 @@
 # TODO выведи данные переменной четыре раза
-A = 9
\ No newline at end of file
+A = 9
+print(A * 4)
```
```suggestion
B = 10
print(B * 2)
```

Nikolay38744 replied:
Согласен с данными изменениями

Constantine7266 replied:
Да согласен

Nikolay38744 committed:
С12.1_1.2py

Изменения в код внес Nik. 
Одобрил Con

Nikolay38744 commented on Стр/10.1/12.1_1.py:
```diff
@@ -1,4 +1,5 @@
 # TODO выведи данные переменной четыре раза
-A = 9
\ No newline at end of file
+B = 10
+print(B * 2)
```
```suggestion
A = "9"
print(A * 4)
```

Nikolay38744 replied:
Con предлагаю внести данные изменения

Constantine7266 committed:
С12.1_1.4py

Изменения предложил Nik.
Зафиксировал создал Commit  Con.

Co-authored-by: Nikolay38744 <97681446+Nikolay38744@users.noreply.github.com>

Nikolay38744 commented on Стр/10.1/12.1_1.py:
```diff
@@ -1,4 +1,5 @@
 # TODO выведи данные переменной четыре раза
-A = 9
\ No newline at end of file
+A = "9"
+print(A * 4)
```
```suggestion
B = "10"
print(B * 2)
```

Constantine7266 committed:
С12.1_1.5py

Изменения в коде предложил Nikolay
Constantine зафиксировал эти изменения commit.

Co-authored-by: Nikolay38744 <97681446+Nikolay38744@users.noreply.github.com>

Nikolay38744 commented on Стр/10.1/12.1_1.py:
```diff
@@ -1,4 +1,5 @@
 # TODO выведи данные переменной четыре раза
-A = 9
\ No newline at end of file
+B = "10"
+print(B * 2)
```
Проверка

Constantine7266 replied:
проверено
